dev/validate_bom__config.py

```diff
@@ -1543,6 +1543,64 @@ def add_files_to_upload(self, options, file_set):
     pass
 
 
+class HaConfigurator(Configurator):
+  """Controls hal config deploy ha (only supported by Kubernetes v2 accounts)."""
```

Comment:
Not sure what this means. Is an account a Spinnaker concept? And the HA configuration is a halyard concept?